pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -218,13 +230,29 @@ def pre_configure_ddp(self):
             )
             self._ddp_kwargs["find_unused_parameters"] = True
 
+    def register_model_hook(self) -> None:
+        if not _TORCH_GREATER_EQUAL_1_7:
+            rank_zero_warn(
+                "Not registering DDP comm hook. "
+                "To use communication hooks, please use PyTorch version at least 1.7.0."
+            )
```

Comment:
this is always going to be called for users below 1.7 which creates noisy logs. we can move the warnings into `register_ddp_comm_hook`
```suggestion
```